api/validator.js

```diff
@@ -44,7 +45,8 @@ const SCHEMA_OUTPUT = Joi.object()
     .keys({
         jobs: SCHEMA_JOBS,
         workflow: Workflow.workflow,
-        errors: Joi.array().items(Joi.string()).optional()
+        errors: Joi.array().items(Joi.string()).optional(),
+        annotations: Annotations.annotations
```

Comment:
It's the one after merged. So `config-parser` will merge share & job level config in the correct order, then put into job permutations.